src/Rox/Core/Consts/PropertyType.php

```diff
@@ -247,6 +247,17 @@ public static function getCustomPropertiesString()
         return self::$custom_properties_string;
     }
 
+    /**
+     * @return PropertyType
+     */
+    public static function getDateTime()
```

Comment:
this is not really a dateTime, it's a `now` string
the name of the custom property is (internal.)`now`
would probably be better to call it `getNowString()` and call the datamember `now_string`